lib/api.js

```diff
@@ -46,6 +71,14 @@ api.getTokens = function() {
   return this.tokenPromise;
 };
 
+/**
+ * Sets this clients refresh token to the value of tokens.refresh_tokens.
```

Comment:
By setting the tokenPromise property it also sets the access_token not just the refresh token